packages/fluentui/react-northstar/src/components/Carousel/Carousel.tsx

```diff
@@ -249,6 +252,8 @@ export const Carousel = (React.forwardRef<HTMLDivElement, CarouselProps>((props,
       },
     },
     mapPropsToBehavior: () => ({
+      paddlePreviousHidden: isPreviousPaddleHidden(),
+      paddleNextHidden: isNextPaddleHidden(),
```

Comment:
nit: what about calling isPreviousPaddleHidden/isNextPaddleHidden only once?
something like:

```tsx
const paddlePreviousHidden = isPreviousPaddleHidden();
    
     // line 254
    mapPropsToBehavior: () => ({
      paddlePreviousHidden,

     // line 436
     hidden: paddlePreviousHidden,
```

Reply:
Or actually... `isPreviousPaddleHidden/isNextPaddleHidden` can be just boolean const instead of function?